app/views/foreman_ansible/job_templates/ansible_windows_updates.erb

```diff
@@ -0,0 +1,187 @@
+<%#
+name: Ansible Windows - Updates
+snippet: false
+template_inputs:
+- name: blacklist
+  required: false
+  input_type: user
+  description: "A list of update titles or KB numbers that can be used to specify
+    which updates are to be excluded from installation.\r\nIf an available update
+    does match one of the entries, then it is skipped and not installed.\r\nEach entry
+    can either be the KB article or Update title as a regex according to the PowerShell
+    regex rules."
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
```

Comment:
I don't think that the resource_type should be set like this.

Reply:
Fixed in second commit